Source/JavaScriptCore/runtime/JSModuleLoader.h

```diff
@@ -47,12 +47,14 @@ class JSModuleLoader final : public JSNonFinalObject {
         return &vm.plainObjectSpace();
     }
 
-    enum Status {
-        Fetch = 1,
-        Instantiate,
-        Satisfy,
-        Link,
-        Ready,
+    enum class Status : uint8_t {
```

Comment:
Maybe we can add `See [[Status]] in https://tc39.es/ecma262/#cyclic-module-record` here?

Reply:
Added.